service/resource/cloudformation/create.go

```diff
@@ -1,7 +1,67 @@
 package cloudformation
 
-import "context"
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	awsCF "github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/giantswarm/aws-operator/service/key"
+	"github.com/giantswarm/microerror"
+)
 
 func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange interface{}) error {
+	customObject, err := key.ToCustomObject(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	createChangeState, ok := createChange.(StackState)
+	if !ok {
+		return microerror.Mask(fmt.Errorf("unexpected type, expecting %T, got %T", createChangeState, createChange))
+	}
+
+	mainTemplate, err := getMainTemplateBody(customObject)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	stackInput := &awsCF.CreateStackInput{
+		StackName:        aws.String(createChangeState.Name),
+		TemplateBody:     aws.String(mainTemplate),
+		TimeoutInMinutes: aws.Int64(30),
```

Comment:
It would be good to have a constant for the timeout.

The 30 minutes worries me. AFAIK the framework only processes a single custom object at a time. So this could block other updates. I'd prefer a lower value and we can increase it later if needed.

Reply:
Good point, I've added the `defaultCreationTimeout` constant to `spec.go`, with value of 10.